src/aims_ui/page_multiple_address.py

```diff
@@ -8,17 +8,46 @@
 from .models.get_endpoints import get_endpoints
 from .models.get_fields import get_fields
 from .models.get_addresses import get_addresses
+from .upload_utils import check_valid_upload
+from .page_error import page_error
 import json
 import csv
 
 page_name = 'multiple_address'
 
-ALLOWED_EXTENSIONS = {'csv'}
 
+def final(searchable_fields,
+          error_description='',
+          error_title='',
+          results_summary_table='',
+          table_results=''):
 
-def allowed_file(filename):
-  return '.' in filename and \
-    filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+  return render_template(
+      f'{page_name}.html',
+      error_description=error_description,
+      error_title=error_title,
+      endpoints=get_endpoints(called_from=page_name),
+      searchable_fields=searchable_fields,
+      table_results=table_results,
+      results_summary_table=results_summary_table,
+      results_page=True,
+  )
+
+
+# In the event of a file being too large, send this custom template
+@app.errorhandler(413)
+def request_entity_too_large(error):
+  # TODO searchable fields unable to set page values to what they were before error
+  searchable_fields = get_fields(page_name)
+
+  for field in searchable_fields:
+    if field.database_name == 'display-type':
+      field.set_radio_status('Download')
+
+  return final(
+      searchable_fields,
+      f'File size is too large. Please enter a file no larger than 2 MB',
```

Comment:
This doesn't need to be an f-string.

Reply:
Ahhh, left over from when it used to be `no larger than {max_file_size} MB`